src/test/java/teammates/test/cases/ui/browsertests/InstructorCommentsPageUiTest.java

```diff
@@ -179,6 +180,7 @@ private void testPanelsCollapseExpand() {
     private void testActions() throws Exception {
         ______TS("action: edit student comment");
         commentsPage.clickStudentCommentEditForRow(1);
+        ThreadHelper.waitFor(1000);
```

Comment:
Fixed time waits like this are not preferred in general. What is this line waiting for?